corehq/apps/callcenter/tasks.py

```diff
@@ -46,6 +47,18 @@ def calculate_indicators():
         indicator_set.get_data()
 
 
+def sync_web_user_usercases_if_applicable(user, spawn_task):
+    if settings.UNIT_TESTING:
+        return
+    for domain in user.get_domains():
+        domain_obj = Domain.get_by_name(domain)
```

Comment:
I think the settings check should move into the loop:

```suggestion
        domain_obj = Domain.get_by_name(domain)
        if not domain_obj and settings.UNIT_TESTING:
            continue
```